app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        person_temp = Person(person["name"], person["age"])
+        person_list.append(person_temp)
+
+    for person in people:
+        if "wife" in person:
+            if person["wife"] is not None:
+                Person.people[person["name"]].wife = \
```

Comment:
You could avoid line breaking in case you will retrieve a person instance at the beginning of the loop instead of referring to `Person.people[person["name"]]` in the if condition

Reply:
def __init__(self, name: str, age: int) -> None:
        self.name = name
        self.age = age
        self.people[name] = self

it's correct?